src/IDAOracle.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity ^0.8.19;
+pragma solidity ^0.8.21;
```

Comment:
Hmmm. Do we want to require this for the interface, vs. a lower version? As changed, it would prevent downstream contracts that use an older compiler version from using the blobstream at all.

Reply:
So we keep solidity versions between the contracts? or we just bump the compilers, but leave all the contracts at 0.8.19?

Reply:
Keep just this specific contract at `0.8.19`, since it's an interface rather than an implementation, IMO. Then bump both the compiler and the version on the other contracts.

Reply:
0908bc7